sound/soc/sof/core.c

```diff
@@ -331,6 +332,13 @@ static int sof_probe_continue(struct snd_sof_dev *sdev)
 	}
 
 	/* boot the firmware */
+
+	/*
+	 * record the jiffies of bootup, send the time stamp between resume
+	 * and bootup can avoid overflow when DSP calculate ms to ticks if
+	 * the time stamp of bootup is a huge number.
+	 */
+	sdev->bootup_timestamp = jiffies_64;
```

Comment:
We need to figure out what time base will be used here. If this REALTIME, MONOTONIC or MONOTONIC_RAW? I suggest the latter so that there is no offset/correction introduced by the kernel. Also is jiffies_64 the way to get the time, or is there a ktime_() function?

Reply:
@plbossart  According to https://elinux.org/Kernel_Timer_Systems I think it is MONOTONIC_RAW. jiffies is common used  in the kernel but maybe we can use ktime_get() and ktime_to_us()?